spec/commands/outdated_spec.rb

```diff
@@ -186,7 +186,7 @@ def test_group_option(group = nil, gems_list_size = 1)
       bundle "outdated --groups"
       expect(out).to include("===== Group default =====")
       expect(out).to include("terranova (newest 9, installed 8, requested = 8)")
-      expect(out).to include("===== Group development, test =====")
+      expect(out).to include("===== Groups development, test =====")
```

Comment:
You will need to add a test for the single non-plural header.

Reply:
line 187?